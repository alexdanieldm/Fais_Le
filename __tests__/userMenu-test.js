/**
 * @format
 */

import 'react-native';
import React from 'react';
import Menu from '../src/screens/userMenu';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('Renders correctly', () => {
	const tree = renderer.create(<Menu />).toJSON();
	expect(tree).toMatchSnapshot();
});
