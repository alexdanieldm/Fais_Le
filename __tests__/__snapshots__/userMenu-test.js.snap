// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0096bd",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "paddingVertical": 25,
        "width": "60%",
      }
    }
  >
    <View>
      <RNSVGSvgView
        align="xMidYMid"
        aria-hidden="true"
        bbHeight={35}
        bbWidth={35}
        data-icon="check-double"
        focusable={false}
        height={35}
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            Object {
              "flex": 0,
              "height": 35,
              "width": 35,
            },
          ]
        }
        vbHeight={512}
        vbWidth={512}
        width={35}
        xmlns="http://www.w3.org/2000/svg"
      >
        <RNSVGGroup>
          <RNSVGPath
            d="M505 174.8l-39.6-39.6c-9.4-9.4-24.6-9.4-33.9 0L192 374.7 80.6 263.2c-9.4-9.4-24.6-9.4-33.9 0L7 302.9c-9.4 9.4-9.4 24.6 0 34L175 505c9.4 9.4 24.6 9.4 33.9 0l296-296.2c9.4-9.5 9.4-24.7.1-34zm-324.3 106c6.2 6.3 16.4 6.3 22.6 0l208-208.2c6.2-6.3 6.2-16.4 0-22.6L366.1 4.7c-6.2-6.3-16.4-6.3-22.6 0L192 156.2l-55.4-55.5c-6.2-6.3-16.4-6.3-22.6 0L68.7 146c-6.2 6.3-6.2 16.4 0 22.6l112 112.2z"
            fill={4294967295}
            propList={
              Array [
                "fill",
              ]
            }
          />
        </RNSVGGroup>
      </RNSVGSvgView>
    </View>
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 30,
          "fontWeight": "bold",
          "marginBottom": 30,
          "marginTop": 10,
        }
      }
    >
      Menu
    </Text>
    <View
      style={
        Object {
          "alignSelf": "stretch",
          "flexDirection": "column",
          "justifyContent": "space-between",
          "marginBottom": 20,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "white",
            "borderColor": "white",
            "borderRadius": 10,
            "flexDirection": "column",
            "justifyContent": "center",
            "marginBottom": 20,
            "paddingHorizontal": 20,
            "paddingVertical": 15,
          }
        }
      >
        <View>
          <RNSVGSvgView
            align="xMidYMid"
            aria-hidden="true"
            bbHeight={40}
            bbWidth={40}
            data-icon="home-alt"
            fill="#0096bd"
            focusable={false}
            height={40}
            meetOrSlice={0}
            minX={0}
            minY={0}
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "borderWidth": 0,
                },
                Object {
                  "flex": 0,
                  "height": 40,
                  "width": 40,
                },
              ]
            }
            vbHeight={512}
            vbWidth={512}
            width={40}
            xmlns="http://www.w3.org/2000/svg"
          >
            <RNSVGGroup
              fill={4278228669}
              propList={
                Array [
                  "fill",
                ]
              }
            >
              <RNSVGPath
                d="M261.56 101.28a8 8 0 00-11.06 0L66.4 277.15a8 8 0 00-2.47 5.79L63.9 448a32 32 0 0032 32H192a16 16 0 0016-16V328a8 8 0 018-8h80a8 8 0 018 8v136a16 16 0 0016 16h96.06a32 32 0 0032-32V282.94a8 8 0 00-2.47-5.79z"
                fill={4278228669}
                propList={
                  Array [
                    "fill",
                  ]
                }
              />
              <RNSVGPath
                d="M490.91 244.15l-74.8-71.56V64a16 16 0 00-16-16h-48a16 16 0 00-16 16v32l-57.92-55.38C272.77 35.14 264.71 32 256 32c-8.68 0-16.72 3.14-22.14 8.63l-212.7 203.5c-6.22 6-7 15.87-1.34 22.37A16 16 0 0043 267.56L250.5 69.28a8 8 0 0111.06 0l207.52 198.28a16 16 0 0022.59-.44c6.14-6.36 5.63-16.86-.76-22.97z"
                fill={4278228669}
                propList={
                  Array [
                    "fill",
                  ]
                }
              />
            </RNSVGGroup>
          </RNSVGSvgView>
        </View>
        <Text
          style={
            Object {
              "color": "#0096bd",
              "fontSize": 13,
              "fontStyle": "italic",
              "fontWeight": "bold",
              "marginTop": 5,
            }
          }
        >
          Go Home
        </Text>
      </View>
      <View
        accessible={true}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "white",
            "borderColor": "white",
            "borderRadius": 10,
            "flexDirection": "column",
            "justifyContent": "center",
            "marginBottom": 20,
            "paddingHorizontal": 20,
            "paddingVertical": 15,
          }
        }
      >
        <View>
          <RNSVGSvgView
            align="xMidYMid"
            aria-hidden="true"
            bbHeight={40}
            bbWidth={40}
            data-icon="trash-alt"
            fill="#0096bd"
            focusable={false}
            height={40}
            meetOrSlice={0}
            minX={0}
            minY={0}
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "borderWidth": 0,
                },
                Object {
                  "flex": 0,
                  "height": 40,
                  "width": 40,
                },
              ]
            }
            vbHeight={512}
            vbWidth={512}
            width={40}
            xmlns="http://www.w3.org/2000/svg"
          >
            <RNSVGGroup
              fill={4278228669}
              propList={
                Array [
                  "fill",
                ]
              }
            >
              <RNSVGPath
                d="M497 273L329 441c-15 15-41 4.5-41-17v-96H152c-13.3 0-24-10.7-24-24v-96c0-13.3 10.7-24 24-24h136V88c0-21.4 25.9-32 41-17l168 168c9.3 9.4 9.3 24.6 0 34zM192 436v-40c0-6.6-5.4-12-12-12H96c-17.7 0-32-14.3-32-32V160c0-17.7 14.3-32 32-32h84c6.6 0 12-5.4 12-12V76c0-6.6-5.4-12-12-12H96c-53 0-96 43-96 96v192c0 53 43 96 96 96h84c6.6 0 12-5.4 12-12z"
                fill={4278228669}
                propList={
                  Array [
                    "fill",
                  ]
                }
              />
            </RNSVGGroup>
          </RNSVGSvgView>
        </View>
        <Text
          style={
            Object {
              "color": "#0096bd",
              "fontSize": 13,
              "fontStyle": "italic",
              "fontWeight": "bold",
              "marginTop": 5,
            }
          }
        >
          Sign Out
        </Text>
      </View>
    </View>
  </View>
</View>
`;
